build.gradle

```diff
@@ -23,3 +23,4 @@ allprojects {
     mavenLocal()
   }
 }
+
```

Comment:
This looks like a mistake. (Or is there really a reason for why it's an improvement to add a blank line here?)